CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Added `abi.type_spec_is_assignable_to` to check for compatible ABI type assignments. ([#540](https://github.com/algorand/pyteal/pull/540))
 
 ## Fixed
+* Subroutines that take ABI type of Transaction now allow any Transaction type to be passed. ([#531](https://github.com/algorand/pyteal/pull/531))
```

Comment:
I think this should go under Changed instead of Fixed

Reply:
change pushed